### PR TITLE
Adding the rm container regression for invalid command

### PR DESCRIPTION
**Pull request by rajanashok**
Create regression test for #2113 (#2123)

### CODE REVIEW

File: tests/test-cases/Group1-Docker-Commands/1-11-Docker-RM.robot [hunk 1]
```diff
@@ -47,6 +47,12 @@ Remove a running container
     ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} rm ${container}
     Should Be Equal As Integers  ${rc}  1
     Should Contain  ${output}  Error response from daemon: You cannot remove a running container. Stop the container before attempting removal or use -f
+
+Remove a container created from unknown executable
+    ${rc}  ${container}=  Run And Return Rc And Output  docker ${params} create busybox xxxx
+    Should Be Equal As Integers  ${rc}  0
```
Comment (mhagen-vmware) [inline tests/test-cases/Group1-Docker-Commands/1-11-Docker-RM.robot]: Shouldn't the rc be !=0?

File: tests/test-cases/Group1-Docker-Commands/1-11-Docker-RM.robot [hunk 2]
```diff
@@ -47,6 +47,12 @@ Remove a running container
...
+    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} rm ${container}
+    Should Be Equal As Integers  ${rc}  0
```
Comment (mhagen-vmware) [inline tests/test-cases/Group1-Docker-Commands/1-11-Docker-RM.robot]: LGTM

File: tests/test-cases/Group1-Docker-Commands/1-11-Docker-RM.md [hunk 3]
```diff
@@ -25,9 +25,10 @@ This test requires that a vSphere server is running and available
 12. Issue docker create --name test busybox to the VIC appliance
 13. Remove the containerVM out-of-band using govc
 14. Issue docker rm test to the VIC appliance
+15. Issue docker rm test to container created from an Unknown executable
```
Comment (anchal-agrawal) [inline tests/test-cases/Group1-Docker-Commands/1-11-Docker-RM.md]: `15. Issue docker rm to container created with an unknown executable`

File: tests/test-cases/Group1-Docker-Commands/1-11-Docker-RM.robot [hunk 4]
```diff
@@ -47,6 +47,12 @@ Remove a running container
     ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} rm ${container}
     Should Be Equal As Integers  ${rc}  1
     Should Contain  ${output}  Error response from daemon: You cannot remove a running container. Stop the container before attempting removal or use -f
+
+Remove a container created from unknown executable
```
Comment (anchal-agrawal) [inline tests/test-cases/Group1-Docker-Commands/1-11-Docker-RM.robot]: Let's move this test to the botton and update its title so we're consistent with `1-11-Docker-RM.md`.

**Comment by anchal-agrawal** [comment]
Minor comments. LGTM

**Comment by mhagen-vmware** [comment]
LGTM

**Comment by rajanashok** [comment]
Rebased

**Pull request merged**